resources/README.md

```diff
@@ -0,0 +1,11 @@
+# Resources
+This directory contains a multitude of language-agnostic resources used in the running of cross-service examples.
+
+These resources are organized in the following sub-directories:
+
+|directory                               |purpose                                                                          |
+|----------------------------------------|---------------------------------------------------------------------------------|
+|[/cdk](/resources/cdk)                  |CDK templates for setting up resources used in cross-service examples.           |
+|[/cfn](/resources/cfn)                  |CloudFormation templates for setting up resources used in cross-service examples.|
+|[/clients](/resources/clients)          |Front-end clients for running cross-service examples.                            |
+|[/sample_files](/resources/sample_files)|Sample files used for cross-service examples.     
```

Comment:
The sample files are used for more than just cross-service examples (like the movie data base used by the basic DynamoDB examples).

Reply:
done!